### ISSUE TITLE
Make more swash letters contextual.

### ISSUE THREAD

**Issue by eclecticfluff** [psb1558/Junicode-font#305]
Since the swash Q already works wonderfully, I would appreciate if all the swash letters could be made to behave in a similar way. It doesn't look good in for lowercase swash letters to appear where the swash would create increased spacing within a word.
![Formed](https://github.com/user-attachments/assets/e3f0f199-fcb8-487d-8bc9-bb76eb97612b)

**Comment by psb1558** [comment]
There are only two lowercase swash letters, e and k. In the next release, these will occur only in word-final position when `swsh` is on:
<img width="670" alt="image" src="https://github.com/user-attachments/assets/5e65d8e5-425e-4bf7-983d-c5f0a8a153d5">
At present the only contextual rules for the various swash Q's are for avoiding collisions, but I've added a rule that cancels any flavor of swash Q in word-final position:
<img width="161" alt="image" src="https://github.com/user-attachments/assets/4250b034-6cec-4713-9996-4a352bcc3008">
Correcting an earlier oversight, I've also added a rule to `swsh` for the extra-long-tail Q:
<img width="615" alt="image" src="https://github.com/user-attachments/assets/dd4824e5-c718-41b8-8a3c-d0a6a2f4c085">
This is in addition to the rules Junicode already has for `cv33`.

Swash uppercase letters other than Q will not be affected by these changes.

For the benefit of casual passers by, I'll just add that the two long-tail Q's are changed to a shorter-tail form when a lowercase letter with descender follows in the second or third position (these are with `swsh[3]`):
<img width="333" alt="image" src="https://github.com/user-attachments/assets/66ce1377-aadf-47b4-9923-bb4d66601293">
